styles/typography.scss

```diff
@@ -2,92 +2,92 @@
 
 @mixin h1-styles {
   font-size: 2.625rem;
-  line-height: 3.15rem;
+  line-height: 1.2;
```

Comment:
Should we just make this a global style?  I've noticed that the designers usually set the same line height for all typography variants.

Reply:
It is, but the global style is 1.4, and this is overriding that for h1s. Perhaps could remove it here... I thought about updating it but who knows what line height will be used next time. 
I figure that as we create more projects form this we can keep tidying up these style files.